torch/csrc/jit/codegen/cuda/type.cpp

```diff
@@ -89,50 +91,109 @@ static const char* expr_type2string(ExprType t) {
   }
 }
 
+bool needFloatSuffix(UnaryOpType t) {
+  switch (t) {
+    case UnaryOpType::Acos:
```

Comment:
wouldn't be easier to list the cases where we _don't_ need the suffix (and return true on the default path) ?

Reply:
Did this somewhere, uncertain if I put it in this PR or the follow ups.